config/application.rb

```diff
@@ -190,6 +190,14 @@ class Application < Rails::Application
     config.logger = Foreman::Logging.logger('app')
     config.active_record.logger = Foreman::Logging.logger('sql')
 
+    if config.serve_static_assets
```

Comment:
For the record, I tried putting this logic in a variety of places with multiple different initializer types and this was the only place that seemed to actually run prior to the middleware stack being built.